services/wallet/api.go

```diff
@@ -6,18 +6,34 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/status-im/status-go/multiaccounts/accounts"
 	"github.com/status-im/status-go/params"
 	"github.com/status-im/status-go/services/wallet/chain"
 	"github.com/status-im/status-go/services/wallet/transfer"
 )
 
 func NewAPI(s *Service) *API {
-	return &API{s}
+	accountDb, _ := accounts.NewDB(s.db)
```

Comment:
Why are we ignoring the error here?

Reply:
Updating by using the accountsdb given by the service